Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -19,6 +19,9 @@ struct CreateBuffersAroundPointsView: View {
     /// The view model for the sample.
     @StateObject private var model = Model()
     
+    /// A Boolean value specifying whether the settings view should be shown
+    @State private var showSettings: Bool = false
```

Comment:
Same for other Booleans below.

```suggestion
    /// A Boolean value indicating whether the settings view is showing.
    @State private var showSettings = false
```